NEWS.md

```diff
@@ -9,6 +9,8 @@ shiny 0.14.0.9000
 
 ### Bug fixes
 
+* Fixed [#1093](https://github.com/rstudio/shiny/issues/1093) better: `updateRadioButtons()` and `updateCheckboxGroupInput()` were not working correctly if the choices were given as a numeric vector. This had been solved in [#1291](https://github.com/rstudio/shiny/pull/1291), but that introduced a different bug [#1396](https://github.com/rstudio/shiny/issues/1396) that this better fix avoids.
```

Comment:
No problem, I just thought we weren't doing that, since only a handful of NEWS entries actually do that. I'll take it this is our new protocol then. I guess it makes sense to do that form now on.